ccd-definition/CaseField.json

```diff
@@ -357,5 +357,14 @@
     "Label": "Date submitted",
     "FieldType": "Date",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "delete",
```

Comment:
This is bad name as it sounds like an action. To follow pattern please change it to be `deletionConsent`.